cypress/integration/cucumber/step_definitions/refapp-3.x/07-logout/logout.js

```diff
@@ -0,0 +1,16 @@
+import {Given} from 'cypress-cucumber-preprocessor/steps';
+
+Given('the user login to the Outpatient Clinic', () => {
+    cy.login();
+    cy.visit('home');
+})
+
+When('the user logout from the application', () => {
+    cy.get('button[name=Users]').click();
+    cy.wait(1000);
```

Comment:
I don't think we need an additional wait because cypress waits for a provided timeout on the configuration.

Reply:
Fixed with [23fde1e](https://github.com/openmrs/openmrs-test-3refapp/pull/11/commits/23fde1e8a61c94e0ca59232b04a0c61aaa786f55)